scripts/nmt/nmt.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+import grpc
+import riva.client.proto.riva_nmt_pb2 as riva_nmt
+import riva.client.proto.riva_nmt_pb2_grpc as riva_nmt_srv
+
+import riva.client
+from riva.client.argparse_utils import add_connection_argparse_parameters
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="Streaming transcription via Riva AI Services")
+    parser.add_argument(
+        "--text", default="mir Das ist mir Wurs, bien ich ein berliner", type=str, help="Text to translate"
+    )
```

Comment:
I suppose it would be better to unite `--text` and `--text-file` into mutually exclusive group.
```suggestion
    inputs = parser.add_mutually_exclusive_group()
    inputs.add_argument(
        "--text", default="mir Das ist mir Wurs, bien ich ein berliner", type=str, help="Text to translate"
    )
    inputs.add_argument("--text-file", type=str, help="Path to file for translation")
```